### PR TITLE
doc: doc fixes related to NCSDK-9787

### PR DESCRIPTION
**Pull request by umapraseeda**
doc fixes related to NCSDK-9787

Signed-off-by: Uma Praseeda <uma.praseeda@nordicsemi.no>

### CODE REVIEW

**Review by greg-fer — APPROVED** [review]

**Review by peknis — APPROVED** [review]

**Review by lemrey — APPROVED** [review]
Changes look good

File: west.yml [hunk 1]
```diff
@@ -114,7 +114,7 @@ manifest:
     - name: nrfxlib
       repo-path: sdk-nrfxlib
       path: nrfxlib
-      revision: 98dc97fe88e22604c165a17ca81e7fbd5d69f82b
+      revision: f0a88d48e4617607bbb9061adac7d5748119ebaa
```
Comment (jfischer-no) [inline west.yml:117]: @umapraseeda can you please include the changes from #5000 in this PR and point revision to https://github.com/nrfconnect/sdk-nrfxlib/commit/1f3ad531bf48d4241b6c60f5c521364a0785bfa7, and close #5000?

Comment (umapraseeda) [reply west.yml]: Done @jfischer-no

**Review by hakonfam — APPROVED** [review]

**Comment by rlubos** [comment]
Added `CI-disable` - the previous CI run failed on unstable tests only, and we need a green CI in order to merge.

**Review by rlubos — APPROVED** [review]

**Pull request merged**